src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -875,9 +875,6 @@ namespace OpenRCT2::Ui::Windows
 
         void RefreshWidgets()
         {
-            pressed_widgets = 0;
-            disabled_widgets = 0;
```

Comment:
This seems a bit unrelated to this change?

Reply:
It is related, it is no longer necessary.